services/configuration/src/tests/mod.rs

```diff
@@ -257,14 +272,19 @@ fn test_apply_with_increased_majority() {
 
 #[test]
 fn test_discard_proposes_with_too_big_majority_count() {
-    let mut testkit: TestKit = TestKit::configuration_default();
+    let mut testkit = TestKitBuilder::validator()
+        .with_validators(4)
+        .with_service(ConfigurationService {
+            config: ConfigurationServiceConfig {
+                majority_count: Some(5),
+            },
+        })
+        .create();
 
     let cfg_change_height = Height(5);
     let new_cfg = {
         let mut cfg = testkit.configuration_change_proposal();
-        let excessive_majority_count = (&testkit.network().validators().len() + 100) as u16;
         cfg.set_service_config("dummy", "First cfg");
-        cfg.set_majority_count(Some(excessive_majority_count));
```

Comment:
this test is probably broken?

We should test `majority_count > .validators().len()`